### DISCUSSION TITLE
Javascript Source?

### DISCUSSION THREAD

**Discussion by empathicqubit** [sonigy/YCS#15] [Q&A]
Hi,

I tend to miss obvious things quite often, so it's possible I've done that here. Do you have Javascript source code in this repo?

Thanks,
EQ

**Comment by StellarSt0rm** [comment]
Hi EQ,

I also have been looking for the JavaScript source, and unfortunately haven't found it either. So, if you're interested in how it fetches the comments (As i found the `youtubei/v1` API to be quite tricky to understand at first), or just want an extension that's a fully open source. Then consider checking out the extension I'm making, it will be fully open source (MIT License)!

It has a style that matches the YouTube web app (at least dark mode, for now) pretty well. It's not yet available, as I'm still making it, but this will be the link once its finished: https://github.com/StellarSt0rm/IYCS
